stack.yaml

```diff
@@ -34,6 +34,14 @@ extra-deps:
 - process-1.6.5.0
 - time-1.8.0.2
 
+- openapi3-3.0.0.1
+- optics-0.3
+- optics-core-0.3.0.1
+- optics-extra-0.3
+- optics-th-0.3.0.2
+- optics-vl-0.2.1
+- insert-ordered-containers-0.2.3.1
```

Comment:
There's a difficult to track down build failure with `0.2.3` (which is the lowest bound for openapi3). It seems it only fails on a certain combination of packages, so I can't provide an upstream PR without further investigation.